l10n_es_aeat_mod347/models/mod347.py

```diff
@@ -347,6 +347,13 @@ def _default_record_id(self):
         default=lambda self: self.env.user,
         copy=False,
     )
+    company_id = fields.Many2one(
```

Comment:
Este campo no debería ser necesario. Para qué lo queréis?

Reply:
Al cargar el external_layout en el informe que se genera se añade en la cabecera la información de la compañía del modelo pero este no tiene, por lo que el informe que aparece para ser enviado a veces puede aparecer con información de una compañía que no es la correcta.

Reply:
OK, gracias por la información. Puedes poner ese comentario encima del campo para que así quede claro y no se vuelva a eliminar en un futuro:

```suggestion
    # Needed for having the field in the external layout rendering scope for catching
    # the proper company for the header/footer
    company_id = fields.Many2one(
```